api/websocket/server/server.go

```diff
@@ -146,13 +146,24 @@ func (ws *WsServer) registryMethod() {
 		// TODO: use signature (or better, with one-time challange) to verify identity
 
 		localNode := s.GetNetNode()
-		addr, pubkey, id, err := localNode.FindWsAddr(clientID)
+
+		isTlsClient := cmd["IsTls"].(bool)
+		var addr, localAddr string
+		var pubkey, id []byte
+
+		if isTlsClient {
+			addr, pubkey, id, err = localNode.FindWssAddr(clientID)
+			localAddr = localNode.GetWssAddr()
+		} else {
+			addr, pubkey, id, err = localNode.FindWsAddr(clientID)
```

Comment:
Could we have others more distinctive API name for FindWsAddr & FindWs**s**Addr ? It is too similar to cause typo.

Reply:
Same as GetWsAddr & localNode.GetWs**s**Addr

Reply:
It's for naming consistent. If we want to change it, we'd better change all the related naming project-wise in another PR

Reply:
👌